tests/test_input.py

```diff
@@ -0,0 +1,28 @@
+"""
+Test cases for the input module.
+"""
+
+import unittest
+import networkx as nx
+from probinet.input.loader import build_adjacency_from_networkx
+from probinet.models.classes import GraphData
+from .fixtures import BaseTest
+
+
+class TestInput(BaseTest):
+
+    def test_build_adjacency_from_networkx(self):
+        G = nx.erdos_renyi_graph(n=14, p=0.5, seed=7, directed=False)
+        # Add weights
+        for i, edge in enumerate(G.edges()):
+            G.edges[edge]["weight"] = 3
+            G.edges[edge]["weight2"] = i
+
+        g_data = build_adjacency_from_networkx(G, weight_list=["weight", "weight2"],file_name=self.folder+"edges.csv")
+
+        self.assertTrue(type(g_data) is GraphData)
```

Comment:
I would probably use `self.assertIsInstance` here, or if you are really worry about the type `self.assertIs` as described in the [documentation](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIsInstance).

Reply:
Great, thanks